io.github.brunoanc.eternalmodmanager.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id" : "io.github.brunoanc.eternalmodmanager",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "46",
+  "sdk" : "org.gnome.Sdk",
+  "sdk-extensions" : [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command" : "eternal_mod_manager",
+  "separate-locales" : false,
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=network",
+    "--device=dri"
+  ],
+  "build-options" : {
+    "append-path" : "/usr/lib/sdk/rust-stable/bin",
+    "env" : {
+        "CARGO_HOME" : "/run/build/eternalmodmanager/cargo"
+    },
+    "build-args" : [
+        "--share=network"
+    ]
+  },
```

Comment:
This won't work on Flathub, builds are done offline. You have to either use a tarball with vendored crates or create a manifest for cargo crates using https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo#flatpak-cargo-generator

Reply:
Fixed.